src/main/kotlin/commands/ForceUpdateCommand.kt

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import Bot
+import Command
+import doesLater
+
+object ForceUpdateCommand : Command("forceupdate"){
+    init{
+        doesLater{
+            try{
+                // TODO: Check member permission
+                if( message.author!!.id != 563138570953687061){ return@doesLater }
+                message.channel.send {
+                    embed {
+                        color = Colors.success
+                        title = "Update success! Check the voice channel."
+                    }
+                }
+            }catch(err: Exception){
```

Comment:
An exception won't be thrown?

Reply:
Here https://github.com/kami-blue/bot-kt/blob/a71b7a7511dbb8b21689affbb5754bf34a14aab4/src/main/kotlin/Main.kt#L203

Reply:
But it's not calling that method...

Reply:
This is calling [updateChannel()](https://github.com/kami-blue/bot-kt/blob/a71b7a7511dbb8b21689affbb5754bf34a14aab4/src/main/kotlin/Main.kt#L148), which is calling [getToken()](https://github.com/kami-blue/bot-kt/blob/a71b7a7511dbb8b21689affbb5754bf34a14aab4/src/main/kotlin/Main.kt#L199) on [this line](https://github.com/kami-blue/bot-kt/blob/a71b7a7511dbb8b21689affbb5754bf34a14aab4/src/main/kotlin/Main.kt#L153) and [this line](https://github.com/kami-blue/bot-kt/blob/a71b7a7511dbb8b21689affbb5754bf34a14aab4/src/main/kotlin/Main.kt#L154)

Reply:
I think you forgot to push because that command only sends a message right now. You shouldn't be relying on exceptions either